### DISCUSSION TITLE
Send alert to deferant channel

### DISCUSSION THREAD

**Discussion by krypto-san** [FabCx/TradingView-Webhook-Bot#63] [Q&A]
Hello 

Is there any way to send the same alert from one TV alert to Two or more channels ?

Thanks

**Comment by fabston** [comment]
Hi @krypto-san,

That is currently not supported. For it to be implemented please open a [feature request](https://github.com/fabston/TradingView-Webhook-Bot/issues/new?assignees=&labels=enhancement&template=feature-request---.md).

Thank you :)

**Reply by krypto-san** [reply]
Thanks for fast reply . your setup works fine . thanks for that too .

**Reply by fabston** [reply]
Your welcome and glad it’s working for you 😊

**Comment by amigolife** [comment]
Hi - new to the script
Whilst this Q&A is from a few years ago - has this feature been upgraded?
ie -  1 webhook from TradingView can send to multiple channels in one go eg Telegram, Discord etc??

Thanks!!!

**Reply by fabston** [reply]
Script works and alerts can be sent to telegram, discord, slack, and email. Due to Elon's changes Twitter / X is broken.

**Reply by amigolife** [reply]
Thanks - what i meant to ask was - it can send to multiple channels simultaneously?
[i think it does] = thanks!
